common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -9,24 +9,27 @@ trait CommercialSwitches {
     SwitchGroup.Commercial,
     "dfp-caching",
     "Have Admin will poll DFP to precache adserving data.",
+    owners = Seq(Owner.withName("commercial team")),
     safeState = On,
     sellByDate = never,
     exposeClientSide = false
   )
 
   val HeaderBiddingUS = Switch(
-     SwitchGroup.Commercial,
-     "header-bidding-us",
-     "Auction adverts on the client before calling DFP (US edition only)",
-     safeState = Off,
-     sellByDate = never,
-     exposeClientSide = true
+    SwitchGroup.Commercial,
+    "header-bidding-us",
+    "Auction adverts on the client before calling DFP (US edition only)",
+    owners = Seq(Owner.withGithub("regiskuckaertz ")),
+    safeState = Off,
+    sellByDate = never,
```

Comment:
good point about the overused thing, we even have a reminder from the venerable nick haley about death by switches and it's becoming true.  The expansion of the switches is probably a proxy for the expansion of the codebase in general though.  That's another PR though.

Also, never used to defined as 31st december 2100 I think, so it was just about possible!  I like the NeverExpiresSwitch idea.